init-tracing-opentelemetry/src/otlp.rs

```diff
@@ -124,6 +138,20 @@ mod tests {
     #[case(Some("http/protobuf"), None, "http/protobuf", "http://localhost:4318")] //Devskim: ignore DS137138
     #[case(Some("grpc"), None, "grpc", "http://localhost:4317")] //Devskim: ignore DS137138
     #[case(None, Some("http://localhost:4317"), "grpc", "http://localhost:4317")] //Devskim: ignore DS137138
+    #[cfg_attr(not(feature = "tls"), ignore)]
+    #[case(
+        None,
+        Some("https://localhost:4317"), //Devskim: ignore DS137138
+        "grpc/tls",
+        "https://localhost:4317" //Devskim: ignore DS137138
+    )]
+    #[cfg_attr(not(feature = "tls"), ignore)]
+    #[case(
+        Some("grpc/tls"),
+        Some("https://localhost:4317"), //Devskim: ignore DS137138
+        "grpc/tls",
+        "https://localhost:4317" //Devskim: ignore DS137138
+    )]
```

Comment:
Could be simplified into :

```
    #[cfg_attr(feature = "tls", case(
        None,
        Some("https://localhost:4317"),
        "grpc/tls",
        "https://localhost:4317"
    ))]
    #[cfg_attr(feature = "tls", case(
        Some("grpc/tls"),
        Some("https://localhost:4317"),
        "grpc/tls",
        "https://localhost:4317"
    ))]
```

- `Devskim: ignore DS137138` is to ignore warning o using `http` instead of `https`
- both annotation cfg_attr & case can be merged (I learn it today)

Reply:
makes sense, corrected